src/BarChart/components/BarChartContainer.ts

```diff
@@ -56,6 +69,9 @@ export default class BarChartContainer extends Component<BarChartContainerProps,
         if (this.subscriptionHandle) {
             mx.data.unsubscribe(this.subscriptionHandle);
         }
+        if (this.intervalHandler) {
+            clearInterval(this.intervalHandler);
+        }
     }
 
```

Comment:
The setInterval is not cleared when the component unmounts.... ??

Reply:
Its called after the component did mount which removes ineterval added